frontend/src/components/Alert.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useRef } from "react";
+import { useEscaper } from "../utils";
+
+/*
+  The Alert component can be filled with a simple string using the content prop. 
+  In that case, the component should be rendered without children.
+  To fill it with more complex JSX, it can be rendered with children.
+  If children are present, the content prop will be ignored.
+*/
+
+export const Alert = ({
```

Comment:
Yes true, Alert maybe is a bit too harsh. I think in that case, Modal would be the proper term, because it's a dialog that blocks the other interactions on the page. (Just learned about that difference.) Would that be acceptable?